lib/streamlit/elements/pyplot.py

```diff
@@ -118,6 +127,7 @@ def marshall(
     image_list_proto: ImageListProto,
     fig: Optional["Figure"] = None,
     clear_figure: Optional[bool] = True,
+    use_container_width: bool = True,
```

Comment:
Should this defaults to `True`?

Reply:
No, because we want to avoid breaking behavior. By default, charts filled the container width and we want to keep this behavior.